spec/models/search/tag_query_spec.rb

```diff
@@ -202,4 +202,43 @@
       expect(q.generated_query[:sort]).to eq([{ "uses" => { order: "asc" } }, { "name.keyword" => { order: "asc" } }, { id: { order: "asc" } }])
     end
   end
+
+  describe "to_wrangle" do
+    let!(:tags) do
+      tags = {
+        used: create(:character, taggings_count_cache: 5),
+        unused_but_canonical: create(:canonical_character),
+        unused_and_not_canonical: create(:character),
+        used_and_canonical: create(:canonical_character, taggings_count_cache: 5),
+        unwrangleable: create(:character, taggings_count_cache: 5, unwrangleable: true),
+        wrangled: create(:canonical_character, common_taggings: [create(:common_tagging)])
+      }
+      run_all_indexing_jobs
+      tags
+    end
+
+    it "returns unwrangled tags in use" do
+      TagQuery.new(to_wrangle: true).search_results.should include(tags[:used])
```

Comment:
Based on RSpec deprecation warnings and https://www.betterspecs.org/#expect, could you rewrite these assertions to use `expect` instead?
```suggestion
      expect(TagQuery.new(to_wrangle: true).search_results).to include(tags[:used])
```